static/js-src/upcoming-page.js

```diff
@@ -33,35 +33,32 @@ async function init() {
   document.body.classList.remove('loading');
 
   channelsArray.forEach((channel) => {
-    channels[channel].components = mapFeaturesToComponents(features[channel].filter(f =>
-      f.browsers.chrome.status.milestone_str === channels[channel].version));
+    channels[channel].components = mapFeaturesToShippingType(features[channel]);
```

Comment:
We no longer need this filter as we are already fetching features according to the milestone

Reply:
looks good.